StoryMap/StoryMap/Screens/Map/Coordinator/MapCoordinator.swift

```diff
@@ -7,62 +7,93 @@
 
 import Foundation
 import UIKit
+import Combine
 
 class MapCoordinator: CoordinatorType {
-    var presenter = UINavigationController()
+	var presenter: UINavigationController
     
-    var addStoryCoordinator: AddStoryCoordinator?
     var storyDetailCoordinator: StoryDetailCoordinator?
     var storyListCoordinator: StoryListCoordinator?
-    
-    var onDidStop: (() -> Void)?
 	
-	private var onDeleteStory: ((Story) -> Void)?
+	private var deleteStorySubject: PassthroughSubject<Story, Never>?
```

Comment:
I suspect the coordinator should not hold a PassthroughSubject.
These should be in ViewModel. This one is not clear for me, if any at all is subscribing to it.